src/store/modules/topic.ts

```diff
@@ -24,6 +27,8 @@ const getters = {
   selectedTopic: (state: State): ITopic | null => state?.selected?.topic,
   selectedTopicCredentialSet: (state: State): ICredentialSet | null =>
     state?.selected?.credentialSet,
+  selectedTopicFullCredentialSet: (state: State): ICredentialSet | null =>
```

Comment:
Is this being referenced anywhere?

Reply:
yes, selectedTopicFullCredentialSet is being referenced in EntityResult.vue